includes/export/abstract-wc-csv-batch-exporter.php

```diff
@@ -46,6 +46,7 @@ public function __construct() {
 		$upload_dir         = wp_upload_dir();
 		$this->file         = trailingslashit( $upload_dir['basedir'] ) . $this->get_filename();
 		$this->column_names = $this->get_default_column_names();
+		apply_filters( 'woocommerce_product_export_init', $this );
```

Comment:
This is filter is not applied in anything here.
Not sure what you indented to do, better you test first.

Reply:
```php
add_filter( 'woocommerce_product_export_init', function ( $exporter ) {
  $exporter->set_limit(100);
});
```

Reply:
An filter need to always return a value. Looks like you want to declare an action...

Reply:
right, didn't thought about that, I'm new to WP.
anyway we changed filter position and now it's returning.